README.md

```diff
@@ -9,3 +9,14 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+To get the mapping to prove isomorphism between graphs, our program will match nodes with the matching degree and ensure the corresponding edges follow the same structure.
```

Comment:
What program? All nodes have matching degrees -- does that mean you can pair any node with any other node?